### PR TITLE
Migrate release notes

### PR DESCRIPTION
**Pull request by edquist**
https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/Release330

- [ ] Enter date into "Migrated" column of google sheet
- [ ] Move TWiki document to the trash

---

https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/Release331

- [ ] Enter date into "Migrated" column of google sheet
- [ ] Move TWiki document to the trash

---

https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/Release3310

- [ ] Enter date into "Migrated" column of google sheet
- [ ] Move TWiki document to the trash

---

https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/Release3311

- [ ] Enter date into "Migrated" column of google sheet
- [ ] Move TWiki document to the trash

---

https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/Release3312

- [ ] Enter date into "Migrated" column of google sheet
- [ ] Move TWiki document to the trash

---

https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/Release3313

- [ ] Enter date into "Migrated" column of google sheet
- [ ] Move TWiki document to the trash

---

https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/Release3314

- [ ] Enter date into "Migrated" column of google sheet
- [ ] Move TWiki document to the trash

---

https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/Release3315

- [ ] Enter date into "Migrated" column of google sheet
- [ ] Move TWiki document to the trash

---

https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/Release3316

- [ ] Enter date into "Migrated" column of google sheet
- [ ] Move TWiki document to the trash

---

https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/Release3317

- [ ] Enter date into "Migrated" column of google sheet
- [ ] Move TWiki document to the trash

---

https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/Release33172

- [ ] Enter date into "Migrated" column of google sheet
- [ ] Move TWiki document to the trash

---

https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/Release3318

- [ ] Enter date into "Migrated" column of google sheet
- [ ] Move TWiki document to the trash

---

https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/Release3319

- [ ] Enter date into "Migrated" column of google sheet
- [ ] Move TWiki document to the trash

---

https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/Release332

- [ ] Enter date into "Migrated" column of google sheet
- [ ] Move TWiki document to the trash

---

https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/Release3320

- [ ] Enter date into "Migrated" column of google sheet
- [ ] Move TWiki document to the trash

---

https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/Release33202

- [ ] Enter date into "Migrated" column of google sheet
- [ ] Move TWiki document to the trash

---

https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/Release3321

- [ ] Enter date into "Migrated" column of google sheet
- [ ] Move TWiki document to the trash

---

https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/Release33212

- [ ] Enter date into "Migrated" column of google sheet
- [ ] Move TWiki document to the trash

---

https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/Release33213

- [ ] Enter date into "Migrated" column of google sheet
- [ ] Move TWiki document to the trash

---

https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/Release3322

- [ ] Enter date into "Migrated" column of google sheet
- [ ] Move TWiki document to the trash

---

https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/Release33222

- [ ] Enter date into "Migrated" column of google sheet
- [ ] Move TWiki document to the trash

---

https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/Release33223

- [ ] Enter date into "Migrated" column of google sheet
- [ ] Move TWiki document to the trash

---

https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/Release3323

- [ ] Enter date into "Migrated" column of google sheet
- [ ] Move TWiki document to the trash

---

https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/Release3324

- [ ] Enter date into "Migrated" column of google sheet
- [ ] Move TWiki document to the trash

---

https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/Release33242

- [ ] Enter date into "Migrated" column of google sheet
- [ ] Move TWiki document to the trash

---

https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/Release3325

- [ ] Enter date into "Migrated" column of google sheet
- [ ] Move TWiki document to the trash

---

https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/Release3326

- [ ] Enter date into "Migrated" column of google sheet
- [ ] Move TWiki document to the trash

---

https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/Release3327

- [ ] Enter date into "Migrated" column of google sheet
- [ ] Move TWiki document to the trash

---

https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/Release333

- [ ] Enter date into "Migrated" column of google sheet
- [ ] Move TWiki document to the trash

---

https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/Release334

- [ ] Enter date into "Migrated" column of google sheet
- [ ] Move TWiki document to the trash

---

https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/Release335

- [ ] Enter date into "Migrated" column of google sheet
- [ ] Move TWiki document to the trash

---

https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/Release336

- [ ] Enter date into "Migrated" column of google sheet
- [ ] Move TWiki document to the trash

---

https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/Release337

- [ ] Enter date into "Migrated" column of google sheet
- [ ] Move TWiki document to the trash

---

https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/Release338

- [ ] Enter date into "Migrated" column of google sheet
- [ ] Move TWiki document to the trash

---

https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/Release339

- [ ] Enter date into "Migrated" column of google sheet
- [ ] Move TWiki document to the trash

---

https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/Release340

- [ ] Enter date into "Migrated" column of google sheet
- [ ] Move TWiki document to the trash

---

https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/Release3402

- [ ] Enter date into "Migrated" column of google sheet
- [ ] Move TWiki document to the trash

---

https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/Release341

- [ ] Enter date into "Migrated" column of google sheet
- [ ] Move TWiki document to the trash

---

https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/Release3412

- [ ] Enter date into "Migrated" column of google sheet
- [ ] Move TWiki document to the trash

---

https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/Release342

- [ ] Enter date into "Migrated" column of google sheet
- [ ] Move TWiki document to the trash

---

https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/Release3422

- [ ] Enter date into "Migrated" column of google sheet
- [ ] Move TWiki document to the trash

---

### CODE REVIEW

File: docs/release/3.4/release-3-4-0-2.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+### Update Software
+
+Once the repositories are installed, you can update to this new release with:
+
+``` console
+[root@client ~] $ yum update
+```
+
+<span class="twiki-macro NOTE"></span> Please be aware that running `yum update` may also update other RPMs. You can exclude packages from being updated using the `--exclude=[package-name or glob]` option for the `yum` command.
```
Comment (bbockelm) [inline docs/release/3.4/release-3-4-0-2.md]: Scrub out all the `twiki-macro` spans.  Above is an example of something that `pandoc` missed, but there's typically one or two per converted document.

**Comment by brianhlin** [comment]
Unsupported release notes are already archived in this repo.

**Comment by brianhlin** [comment]
I've added the release notes page and the ToC-hiding code (oh god it's hacky). We still need to migrate these two new docs:

- https://twiki.grid.iu.edu/bin/view/Documentation/Release3/Release3328
- https://twiki.grid.iu.edu/bin/view/Documentation/Release3/Release343

**Comment by brianhlin** [comment]
@edquist could you convert those two docs and add them to this PR?

**Comment by brianhlin** [comment]
@timtheisen just a heads up of what the docs are looking like for the release notes. The October release notes will have to be in markdown

**Comment by brianhlin** [comment]
And the other checkboxes:

https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/ReleaseNotes

- [ ] Enter date into "Migrated" column of google sheet
- [ ] Move TWiki document to the trash

---

https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/Release3328

- [ ] Enter date into "Migrated" column of google sheet
- [ ] Move TWiki document to the trash

---

https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/Release343

- [ ] Enter date into "Migrated" column of google sheet
- [ ] Move TWiki document to the trash

**Pull request merged**